features/org.palladiosimulator.simulizar.test.commons.feature/feature.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="org.palladiosimulator.simulizar.test.commons.feature"
+      label="Palladio Component Model - Analyzer: SimuLizar - Test Commons"
+      version="5.0.0.qualifier"
+      plugin="org.palladiosimulator.branding"
+      license-feature="org.palladiosimulator.license"
```

Comment:
I would suggest using the more recent org.palladiosimulator.license.epl2. As I think, we should use (and mostly are using) the EPLv2 for new code.